src/main/java/com/timgroup/statsd/Telemetry.java

```diff
@@ -29,8 +38,15 @@
     protected final String packetsDroppedMetric = "datadog.dogstatsd.client.packets_dropped";
     protected final String packetsDroppedQueueMetric = "datadog.dogstatsd.client.packets_dropped_queue";
     protected final String aggregatedContextsMetric = "datadog.dogstatsd.client.aggregated_context";
+    protected final String aggregatedContextsByTypeMetric = "datadog.dogstatsd.client.aggregated_context_by_type";
+
+    protected final String aggregatedGaugeContextsMetric = "datadog.dogstatsd.client.aggregated_context_gauge";
```

Comment:
it's `aggregated_context_by_type` (https://github.com/DataDog/datadog-go/blob/master/statsd/telemetry.go#L140) which is also tagged with the `metric_type` tag now.

Reply:
Yeah, that code was dead code, we had already fixed that. Thanks for spotting it.